### ISSUE TITLE
Keyboard navigation is awkward and tedious

### ISSUE THREAD

**Issue by sdellis** [pulibrary/pulfalight#498]
See Amazon's approach for reference example.

**Facets should receive focus before search results when navigating via keyboard.** 

Severity: Confusing or Annoying
WCAG Guideline:
2.4.3 A: Interactive controls receive focus in an order that makes sense. https://www.w3.org/WAI/WCAG21/quickref/#focus-order 

**Use "skip to" links**
There should also be a "skip to search results" link that appears when the **facets block** gains focus and a "Skip to description and media" when the ToC gains focus, similar to "skip to main content" at the top of the page. All "skip" links should be visible when in focus.

Severity: Confusing or Annoying
WCAG Guideline:
2.4.1 A: Users can bypass blocks of content that are repeated on multiple Web pages, such as navigation menus (skip links, headings, toggles). 
https://www.w3.org/WAI/WCAG21/quickref/#bypass-blocks
